openmdao/core/system.py

```diff
@@ -2116,6 +2121,9 @@ def add_response(self, name, type_, lower=None, upper=None, equals=None,
             Mapping of this response to each design variable where simultaneous derivs will
             be used.  Each design variable entry is another dict keyed on color, and the values
             in the color dict are tuples of the form (resp_idxs, color_idxs).
+        cache_linear_solution : bool
```

Comment:
I think this description is a little unclear (plus this PR is missing documentation.) 

I gather the purpose of this feature is to start the linear solution with an initial guess that is set to the cahced values of the previous linear solve.